app/views/sign_up/completions/show.html.erb

```diff
@@ -24,6 +24,15 @@
         sp: content_tag(:strong, decorated_session.sp_name),
       ) %>
 </p>
+<% if @multiple_factors_enabled == false && IdentityConfig.store.select_multiple_mfa_options == false %>
```

Comment:
Should we be checking that `select_multiple_mfa_options` is _enabled_, not disabled? I'm expecting the notice would only be shown when feature flag is turned on?

```suggestion
<% if !@multiple_factors_enabled && IdentityConfig.store.select_multiple_mfa_options %>
```

Reply:
It should be enabled because we only want it when `select_multiple_mfa_options` is true. I think I just needed fresh eyes on this 😅